lib/sig.js

```diff
@@ -113,7 +113,7 @@ var actionsObjRegexp = new RegExp(
     jsKeyStr + sliceStr   + '|' +
     jsKeyStr + spliceStr  + '|' +
     jsKeyStr + swapStr +
-  '),?\\n?)+)\\};'
+  '),?(\\r\\n|\\r|\\n)?)+)\\};'
```

Comment:
Yes but, wouldn't `\\r?\\n?` also match that string? Could you test that on Windows?